Sources/Components/Radio/Radio.swift

```diff
@@ -0,0 +1,136 @@
+import SwiftUI
+
+extension Warp {
+    /// Radio buttons allow users to select one option from a set.
+    ///
+    /// `Radio` is a customizable component that allows users
+    /// to create a list of radio buttons where only one can be selected at any given time.
+    ///
+    /// - Parameters:
+    ///   - selectedOption: A binding to the currently selected option.
+    ///   - options: An array of options that conform to `Identifiable` and `Hashable`.
+    ///   - label: A closure that provides a label for each option.
+    ///   - state: The state of the radio button group (default, error, disabled).
+    public struct Radio<Option: Identifiable & Hashable>: View {
```

Comment:
```suggestion
    public struct RadioButtonGroup<Option: Identifiable & Hashable>: View {
```

It seems you had your mind set on RadioButtonGroup previously in a comment:

> A single radio button view used within `RadioButtonGroup`.

And I like that better actually. That or at least something that conveys that this component is not a single radio button...it is a list of radio buttons 😄